### DISCUSSION TITLE
raw_ahrefs.py Example not working

### DISCUSSION THREAD

**Discussion by ghost** [seleniumbase/SeleniumBase#2942] [General]
https://github.com/seleniumbase/SeleniumBase/blob/master/examples/raw_ahrefs.py

seleniumbase.common.exceptions.NoSuchElementException: Message: 
 Element {p:contains("github.com/seleniumbase/SeleniumBase")} was not present after 7 seconds!
===================== {2captcha.py:3:SB} failed in 15.11s ======================

**Comment by mdmintz** [comment]
The script is working for me. If it's not working for you, then results may be different for a different geolocation.

**Comment by ghost** [comment]
I forgot to mention I'm on linux, thus using the xvfb option.
Now on SB 4.29.2

    with SB(uc=True, test=True, xvfb=True) as sb:

and 

        sb.uc_gui_click_cf(frame="iframe", retry=True)

I got it to work but with the last CF update it not solving.

**Reply by mdmintz** [reply]
Upgrade to `4.29.3` or newer if available. See https://github.com/seleniumbase/SeleniumBase/issues/2966#issue-2432599667 for details.
The example test was also updated: [SeleniumBase/examples/raw_ahrefs.py](https://github.com/seleniumbase/SeleniumBase/blob/master/examples/raw_ahrefs.py).
